src/Compilers/CSharp/Portable/Symbols/Synthesized/GeneratedNameKind.cs

```diff
@@ -35,6 +35,8 @@ internal enum GeneratedNameKind
         AsyncBuilderField = 't',
         DynamicCallSiteContainerType = 'o',
         DynamicCallSiteField = 'p',
+        MethodGroupConversionCacheFrame = 'q',
+        MethodGroupConversionCacheDelegateField = 'r',
```

Comment:
This is an exhaustive list of the kinds emitted by Roslyn, but I'm concerned that dev12 may have emitted additional kinds, possibly including 'q' and 'r'.